dir.traversal.targets

```diff
@@ -99,5 +99,8 @@
   <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
 
   <Target Name="RestorePackages" DependsOnTargets="$(TraversalRestorePackagesDependsOn)" />
+  <Target Name="BuildAndTest" DependsOnTargets="Build;Test" />
```

Comment:
These were copied from <root>\dirs.targets as we needed them for our new traversal .builds projects.